main/modes/games/ultimateTTT/ultimateTTTcpuPlayer.c

```diff
@@ -316,21 +332,85 @@ static bool selectSubgame_hard(ultimateTTT_t* ttt, int* x, int* y)
     // handle a tie??? Just use the TIE player? Yeah probably)
     // - We calculate the next possible move on that board, and then take a look at the board we'd need to win to get
     // our marker there
-    // - So... I guess we should take all the possible moves in that board, and then what...
+    // - So... I guess we should take all the possiblebest overal moves in that board, and then what...
+
+    // Who are we?
```

Comment:
Still figuring that out myself

Reply:
In fairness, has anyone really been far even as decided to use even go want to do look more like possiblebest overal moves in that board? and forgotten who we really are?